pkg/app/app.go

```diff
@@ -35,6 +35,7 @@ import (
 
 	"os"
 
+	"github.com/fsouza/go-dockerclient/external/github.com/Sirupsen/logrus"
```

Comment:
Whoops, never realized that `logrus` was aliased to `log`, so my editor ended up importing this. Fixed! Nice catch, man :)